aiida/tools/importexport/dbexport/__init__.py

```diff
@@ -448,29 +634,83 @@ def export_tree(
         'input': node_pk_2_uuid_mapping[link.source_id],
         'output': node_pk_2_uuid_mapping[link.target_id],
         'label': link.link_label,
-        'type': link.link_type
+        'type': link.link_type,
     } for link in traverse_output['links']]
 
     progress_bar.update()
 
+    return (
+        node_ids_to_be_exported,
+        node_pk_2_uuid_mapping,
+        links_uuid,
+        graph_traversal_rules,
+    )
+
+
+def check_node_licenses(
+    node_ids_to_be_exported: Set[int],
+    allowed_licenses: Optional[Union[list, Callable]],
+    forbidden_licenses: Optional[Union[list, Callable]],
+) -> None:
+    # TODO (Spyros) To see better! Especially for functional licenses
```

Comment:
no docstring... I see pylint is very lenient in this file... ;-)

Reply:
Well I didn't change the pylint rules that were already there 😉

Reply:
actually I don't think pylint checks for docstrings anyway, because there nothing disabling it